runtime/moonbeam/src/lib.rs

```diff
@@ -1012,13 +1013,33 @@ impl pallet_proxy::Config for Runtime {
 	type AnnouncementDepositFactor = ConstU128<{ currency::deposit(0, 56) }>;
 }
 
+use pallet_migrations::{GetMigrations, Migration};
+pub struct TransactorRelayIndexMigration<Runtime>(sp_std::marker::PhantomData<Runtime>);
+
+impl<Runtime> GetMigrations for TransactorRelayIndexMigration<Runtime>
+where
```

Comment:
Not ideal as all migrations are tracked in the  runtime common file...but I guess this migrations has some different stuff as it needs different inputs for all runtimes. Just remember to add a comment that it was passed in the runtime-xx

Reply:
Yes, we had better not move them for now.